floaters/src/main/java/com/mdg/droiders/floaters/FloatingViewContainer.java

```diff
@@ -75,4 +80,57 @@ View getCollapsedView() {
     View getExpandedView() {
         return expandedView;
     }
+
+    /**
+     * Init on click listener in the MUSIC PLAYER expanded version of the floating view.
+     *
+     * @param context The @{@link Context} to use for displaying toast on view click
+     */
+    private void initOnClickListeners(final Context context) {
+
+        ImageView playButton = (ImageView) mFloatingView.findViewById(R.id.play_btn);
```

Comment:
\[Checkstyle\] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4\.